docs/examples/plot_rainbowgram.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+"""
+================
+Rainbowgrams
+================
+This notebook demonstrates how to use "Rainbowgrams" to simultaneously 
+visualize amplitude and (unwrapped) phase (differential) as demonstrated in the
+`NSynth paper <https://proceedings.mlr.press/v70/engel17a/engel17a.pdf>`
+"""
+# Code source: Brian McFee
+# License: ISC
+
+#########################
+# Standard imports
+import numpy as np
+import matplotlib.pyplot as plt
+import librosa
+
+import librosa.display
+
+############################################# 
+# We implemented a stft method to visualize the rainbowgram and demonstrated the result with a chirp signal.
+# A chirp signal starts at a low frequency and gradually increases in frequency over time. We then separated the magnitude and phase components of the signal
+sr = 22050
+y = librosa.chirp(fmin=32, fmax=32 * 2**5, sr=sr, duration=10, linear=True)
+D = librosa.stft(y)
+mag, phase = librosa.magphase(D)
+
+###########################################
+# We should be visualizing the demodulated phase differential derived by subtracting 2π*f*t 
+# from each phase estimate prior to unwrapping, where f and t are the frequency and time.
+freqs = librosa.fft_frequencies()
+times = librosa.times_like(D)
+
+phase_exp = 2*np.pi*np.multiply.outer(freqs,times)
+####################
+# Plot the spectrum.
+fig, ax = plt.subplots()
+img = librosa.display.specshow(np.diff(np.unwrap(np.angle(phase)-phase_exp, axis=1), axis=1, prepend=0),
+                         cmap='hsv', 
+                         alpha=librosa.amplitude_to_db(mag, ref=np.max)/80 + 1,
+                         ax=ax,
+                         y_axis='log', 
+                         x_axis='time')
+ax.set_facecolor('#000')
+fig.colorbar(img, ax=ax)
```

Comment:
You have two colorbars here, resulting in 
![image](https://user-images.githubusercontent.com/1190540/207586738-9c1e00d2-df74-48bb-b3c8-8a191010bc1f.png)